webapp/src/mutator.ts

```diff
@@ -138,7 +138,7 @@ class Mutator {
         )
     }
 
-    async setDefaultTemplate(blockId: string, oldTemplateId: string, templateId?: string, description = 'set default template') {
+    async setDefaultTemplate(blockId: string, oldTemplateId: string, templateId = '', description = 'set default template') {
```

Comment:
I think adding the default value to `''` is error prone. It is easy to think that the `setDefaultTemplate(blockId, newTemplateId)` is correct, and is not. I prefer to have it explicitly shown everywhere, or even have a `clearDefaultTemplate`.

Reply:
You're right. Let me split this method

Reply:
done